### PR TITLE
clock: avoid deprecated g_type_class_add_private

### PR DESCRIPTION
**Pull request by yetist**

### CODE REVIEW

File: applets/clock/calendar-window.c [hunk 1]
```diff
@@ -475,7 +471,7 @@ calendar_window_init (CalendarWindow *calwin)
 {
 	GtkWindow *window;
 
-	calwin->priv = CALENDAR_WINDOW_GET_PRIVATE (calwin);
+    calwin->priv = calendar_window_get_instance_private (calwin);
```
Comment (sc0w) [inline applets/clock/calendar-window.c]: wrong indentation

File: applets/clock/clock-face.c [hunk 2]
```diff
@@ -407,11 +403,11 @@ remove_pixbuf_from_cache (const char *key,
...
-        ClockFacePrivate *priv = CLOCK_FACE_GET_PRIVATE (this);
-	const gchar *size_string[2] = { "small", "large" };
-        const gchar *daytime_string[4] = { "morning", "day", "evening", "night" };
-	gchar *cache_name;
-	gchar *name;
+    ClockFacePrivate *priv = clock_face_get_instance_private (this);
+    const gchar *size_string[2] = { "small", "large" };
+    const gchar *daytime_string[4] = { "morning", "day", "evening", "night" };
+    gchar *cache_name;
+    gchar *name;
```
Comment (sc0w) [inline applets/clock/clock-face.c]: wrong indentation

**Comment by lukefromdc** [comment]
I am getting a segfault on adding a location to the calendar, but I am also getting it with master so not related to this PR. A  number of mate applications have acted up since recent changes in Debian Unstable, some just having to be rebuilt but not this one as the PR test shows. Looking for the issue now, will open an issue when I find the problem under gdb

**Comment by yetist** [comment]
Maybe I should do the code indent first, and then make the real changes?

**Comment by lukefromdc** [comment]
Rebased after latest change in master, fixed indent and force-pushed

File: applets/clock/clock-face.c (hunk 2 above)
Comment (lukefromdc) [reply applets/clock/clock-face.c]: Fixed and force-pushed to keep things moving

Comment (sc0w) [reply applets/clock/clock-face.c]: no, it isn't fixed

see now in master https://github.com/mate-desktop/mate-panel/blob/master/applets/clock/clock-face.c#L410

Comment (raveit65) [reply applets/clock/clock-face.c]: fixed by https://github.com/mate-desktop/mate-panel/pull/1018

**Pull request merged**